pipelines/batch/src/main/java/org/openmrs/analytics/FhirEtl.java

```diff
@@ -82,8 +83,8 @@ static OpenmrsUtil createOpenmrsUtil(FhirEtlOptions options, FhirContext fhirCon
         fhirContext);
   }
 
-  private static Schema getSchema(String resourceType) {
-    ParquetUtil parquetUtil = new ParquetUtil(null); // This is used only to get schema.
+  private static Schema getSchema(String resourceType, FhirVersionEnum version) {
+    ParquetUtil parquetUtil = new ParquetUtil(version, null); // just for the schema.
```

Comment:
nit: how about we delete the comment as the function call after does what the comment say?

Reply:
Done.